src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
+const rounder = (num: number): number => {
+  return Math.floor(num * 100) / 100;
+};
 
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
```suggestion
  color: TColor;
```
Consider defining a type for declaring available colors

Reply:
agree, please do it